ui/components/institutional/institutional-entity-done-page/institutional-entity-done-page.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import Button from '../../ui/button';
+import { getMostRecentOverviewPage } from '../../../ducks/history/history';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+
+export default function InstitutionalEntityDonePage(props) {
+  const mostRecentOverviewPage = useSelector(getMostRecentOverviewPage);
+  const t = useI18nContext();
+  const { history, location } = props;
+  const { state } = location;
+
+  return (
+    <div className="page-container institutional-entity__container">
+      <div className="page-container__content">
+        <div className="institutional-entity__form">
+          <div className="entity-connect__compliance-activated">
+            <img
+              className="entity-connect__compliance-activated__img"
+              src={state.imgSrc}
+              alt="Entity image"
+            />
+            <h4 className="entity-connect__header__title">{state.title}</h4>
+            <p className="entity-connect__header__msg">{state.description}</p>
+          </div>
+        </div>
+      </div>
+      <div className="page-container__footer">
+        <footer>
+          <Button
+            type="primary"
+            large
+            className="page-container__footer-button"
+            data-testid="click-most-recent-overview-page"
+            onClick={() => history.push(mostRecentOverviewPage)}
+          >
+            {t('close')}
+          </Button>
+        </footer>
+      </div>
+    </div>
```

Comment:
I don't see any styles in this folder for these classes, and i would like to see the Box and Text components adopted here.

Reply:
@brad-decker can you post here an example of a working component using Text? I'm having problems adding it. Thanks

Reply:
@brad-decker could you unlock the PR? All the changes have been made